.travis.yml

```diff
@@ -25,7 +28,7 @@ script:
 - python test/testsuite.py
 deploy:
   provider: pypi
-  user: sunng
+  username: sunng
```

Comment:
Why did you change user to username? This is not the way it's described in https://docs.travis-ci.com/user/deployment/pypi/

Reply:
Thx... travis-ci/docs-travis-ci-com#2803